shell/models/management.cattle.io.setting.js

```diff
@@ -12,7 +12,7 @@ export default class Setting extends HybridModel {
     let out = super._availableActions;
 
     // Some settings are not editable
-    if ( settingMetadata?.readOnly || this.fromEnv ) {
```

Comment:
Are there possible regressions here ? What is the function of `fromEnv` ?

Reply:
It's mentioned in the PR: it's meant to avoid changes in the values which are environment variable. At the current state there's none except this case and it has been changed by the server side during development phase (4 months 😬).

Reply:
As extra point, this has been asked during the daily and nobody was against.

Reply:
Ok  sorry, I didn't notice the mention. I think the serve side change is enough to be confident.